Dependencies/CMakeLists.txt

```diff
@@ -11,6 +11,10 @@ if(WINDOWS_STORE)
     set_property(TARGET arcana PROPERTY FOLDER Dependencies)
 endif()
 
+# -------------------------------- GSL -----------------------------------
+# Dependencies: none
+add_subdirectory(arcana.cpp/Source/Submodules/GSL)
```

Comment:
why do you need this?

Reply:
I'm using gsl::finally in xr.cpp to revert OpenGL state changes I make since we are taking over the gl context every frame (sharing it with Babylon rendering). Ideally we would use our own context that wraps the context used by Babylon rendering, but I never was able to get this to work.

Reply:
Okay. I'm a bit concerned about reaching into a dependency's dependency. Not sure what the implications are.